### PR TITLE
Codec Readme

### PR DESCRIPTION
**Pull request by nplasterer**
As discussed it would be helpful to have more documentation around using content types and creating content types. This takes a first pass at that.

### CODE REVIEW

File: library/src/main/java/org/xmtp/android/library/codecs/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Adding specific content types to your application
+
+You can create any content type you want to be used in your application you can read me about creating content types here: [LINK]
+
+We have a few content types backed into the sdk directly for easy consumption (encryption and decryption) across applications.
+
+TextCodec - (always registered by default) allows the passing of plain text.
+AttachmentCodec - allows the passing of attachments
+RemoteAttachmentCodec - allows the passing of remote attachments
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/codecs/README.md:9]: I think we should make this a table that we update as we add more SDK supported content types.

File: library/src/main/java/org/xmtp/android/library/codecs/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```kotlin
+val newConversation = client.conversations.newConversation(walletAddress)
+
+newConversation.send(
+    content = remoteAttachment,
+    options = SendOptions(contentType = ContentTypeRemoteAttachment),
+)
+```
+
+### Receive a remote attachment
```
Comment (jhaaaa) [inline library/src/main/java/org/xmtp/android/library/codecs/README.md:65]: In this step, are we also decrypting and decoding? =)

Comment (nplasterer) [reply library/src/main/java/org/xmtp/android/library/codecs/README.md:65]: Yes!

**Comment by nplasterer** [comment]
@nakajima if you get a moment can you read over this doc and make sure this all makes sense?

**Review by nakajima — COMMENTED** [review]
Everything makes sense to me here.

**Review by jhaaaa — APPROVED** [review]
Thank you for this @nplasterer!

**Pull request merged**